cmd/dextre/cmd/robots_gen_gc.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// robotsGenGCCmd represents the robots gen_gc command
+var robotsGenGCCmd = &cobra.Command{
+	Use:   "gen_gc",
+	Short: "Get golden copy for robots check",
+	Long: `Gets the specified URL contents and stores it as a golden copy
+for subsequent robots checks.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if goldenCopy == "" {
+			log.Fatal("golden copy is mandatory")
+		}
+		if len(args) < 1 {
+			log.Fatal("No URL specified")
+		}
+		url := args[0]
+
+		if _, err := os.Stat(goldenCopy); err == nil {
+			log.Fatal("Golden copy file already exists")
```

Comment:
Wouldn't it be better to have a more descriptive message? 
`log.Printf("Could not create file %s: %v\n", goldenCopy, err)`

Reply:
Well, this is not creating the file, it is just checking the file exists.

Reply:
Yep, my mistake. But the point was telling the user where we tried to access:
`log.Fatalf("Golden file already exists %s: %v\n", goldenCopy, err)`